iroh/src/commands.rs

```diff
@@ -567,6 +567,27 @@ pub enum BlobCommands {
     /// Delete content on the node.
     #[clap(subcommand)]
     Delete(self::delete::Commands),
+    /// Get a ticket to share this blob.
+    Share {
+        /// Hash of the blob to share.
+        hash: Hash,
+        /// Add this token to the ticket.
+        #[clap(long)]
+        token: Option<String>,
+        /// Do not include derp reion information in the ticket.
+        #[clap(long, conflicts_with = "derp_only", default_value_t = false)]
+        no_derp: bool,
+        /// Include only the derp region information in the ticket.
+        #[clap(long, conflicts_with = "no_derp", default_value_t = false)]
+        derp_only: bool,
+        /// Create a ticket for the blob only. If the blob is a collection, the requester will not
+        /// download it recursively.
```

Comment:
```suggestion
        /// Create a ticket for the blob only. If the blob is a collection, the requester only
        /// fetch the collection list
```

Reply:
done in [f4cbea3](https://github.com/n0-computer/iroh/pull/1746/commits/f4cbea3492b16b9de1bb4a9fe95ef7b181aedbf6)